### PR TITLE
Feedback sws master volume

### LINKED ISSUE #470: Is it possible to give feedback to these SWS actions for the volume control of the master track?
Hi,
I found 3 SWS actions to control the master volume of a DB or to reset to 0 db in the master track.
The actions are:
Xenakios / SWS: Nudge master volume 1 dB up
Xenakios / SWS: Nudge master volume 1 dB down
To reset to 0 DB:
Xenakios / SWS: Set master volume to 0 dB
Is it possible that OSARA can give feedback for these actions?
Greetings!

### PR DESCRIPTION
**Pull request by ScottChesworth**
Resolves #470

### CODE REVIEW

**Comment by ScottChesworth** [comment]
Hmmm, All checks have failed. Did I goof @jcsteh?

**Comment by jcsteh** [comment]
The problem is there are merge conflicts because this isn't based off latest master. I wonder whether your local master is a bit of a mess now? Maybe you accidentally committed some stuff to master at some point.

I'll rebase this for you, but I'd suggest doing this to clean up master:
git checkout master
git fetch
git reset --hard origin/master
That'll get your master back to the upstream master.

**Comment by ScottChesworth** [comment]
> I wonder whether your local master is a bit of a mess now?
> 
That's plausable. I may even go so far as likely, lol.
I did git pull before creating the branch and adding to reaper_osara.cpp earlier, but guess that's not enough. Should I be hard resetting master before creating new branches off of it each time?

**Comment by jcsteh** [comment]
You shouldn't need to reset every time, no. What I think has happened is that at some point, you accidentally committed something to master instead of a branch. It's easy to do; I do it all the time myself. The key is remembering to branch before you commit, but it's hard to remember when you're like "oo, I got it working, now I want to save it". :)

**Comment by ScottChesworth** [comment]
I usually create a new branch before I start twiddling with stuff here, just in case I need to pretend the last thing I tried never happened lol. In any case, all reset now, so hopefully this won't happen in the next PR (there'll be a couple of key map fixes coming up before I turn in).

File: readme.md [hunk 1]
```diff
@@ -108,6 +108,9 @@ Most of these are actions built into REAPER, but a few are very useful actions f
 - (SWS extension) Xenakios/SWS: Nudge volume of selected tracks down: Alt+Shift+DownArrow
 - Track: Nudge master track volume up: Shift+F12
 - Track: Nudge master track volume down: Shift+F11
+- Xenakios/SWS: Nudge master volume 1 dB up: 
+- Xenakios/SWS: Nudge master volume 1 dB down: 
+- Xenakios/SWS: Set master volume to 0 dB:
```
Comment (jcsteh) [inline readme.md:113]: nit: No point in having the trailing colons until we have bindings for these.
```suggestion
- Xenakios/SWS: Nudge master volume 1 dB up
- Xenakios/SWS: Nudge master volume 1 dB down
- Xenakios/SWS: Set master volume to 0 dB
```

**Pull request merged**